docs/modules/ROOT/pages/migration_guide.adoc

```diff
@@ -1,9 +1,9 @@
-= Devfile v2 Migration Guide
+= Migrating from devfile v1 to devfile v2
```

Comment:
Unless there's a specific reason not to, I believe the use of `v1` vs `v1.0` and `v2` vs `v2.0` should be unified.